gubernator/templates/pr_dashboard.html

```diff
@@ -40,10 +45,21 @@ <h3 class="pr-section">
 			<td>{{pl['assignees']|join(", ")}}
 			<td>{{pl|classify_size}}
 			<td>{{pl['title']}}
+			% if title == "Needs Attention" and acks is not none
+				% set latest = pl|get_latest(user)
+				% if latest
+					<td><button onclick="ack(event, '{{pr.repo}}', {{pr.number}}, {{latest}});">ACK</button>
+				% endif
+			% endif
 		% endfor
 	</table>
 	% else
 		<p><h4 class="pr-section">No Results</h4>
 	% endif
 % endfor
+
+<div id="help">
+<p><b>Needs Attention</b> is a heuristic grouping based on a <a href="https://github.com/kubernetes/test-infra/blob/master/gubernator/github/classifier.py#L288">simple state machine</a>. Use the "ACK" button to to remove a PR from "Needs Attention" until something new occurs.
```

Comment:
> Use the "ACK" button to to remove a PR from "Needs Attention"

![image](https://cloud.githubusercontent.com/assets/1929644/22301876/5f42d91e-e2e1-11e6-939c-30b978ae2b01.png)

Dear Ryan, there are too many "to". Please remove one.